src/CMakeLists.txt

```diff
@@ -379,6 +380,7 @@ add_subdirectory(private-frameworks)
 add_subdirectory(external/usertemplate)
 add_subdirectory(AVKit)
 add_subdirectory(MultipeerConnectivity)
+add_subdirectory(CoreGraphics)
```

Comment:
~~We have CoreGraphics in src/external/cocotron/CoreGraphics~~ seems like this is already taken care of